actions/rules_enable.py

```diff
@@ -0,0 +1,18 @@
+from lib.action import St2BaseAction
+from lib.formatters import format_rule_update_result
+
+__all__ = [
+    'St2RulesEnableAction'
+]
+
+
+class St2RulesEnableAction(St2BaseAction):
+    def run(self, pack=None, name=None, exclude=None):
+
+        rule = self._manipulate_rule(pack=pack, name=name, enabled=True)
```

Comment:
This code looks shared between the disable action. Consider putting it into a common function

Reply:
You mean make a single action to enable/disable? Because I already created the `_manipulate_rule()` method, since they share it.

Reply:
the following code is copy/pasted between the two actions:

```
        rule = self._manipulate_rule(pack=pack, name=name, enabled=False)
        if rule is None or isinstance(rule, str):
            # error happened
            return False, rule
        else:
            # all good here
            return True, format_rule_update_result(rule, exclude)
```
The only difference is `enabled=True` or `enabled=False`.

I would like the duplicated code collapsed into a common function, right now the duplicated code is around the handling of the result from the `manipulate_rule` function. I see two possible options:

1. Change the `manipulate_rule` function to just return what you should actually be returning from the action.
2. Create a new function that takes the result of `manipulate_rule` and returns the result for the action.

Reply:
OK. Will refactor this.